satellite/satellitedb/satellitedbtest/bandwidthagreement_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedbtest
```

Comment:
These tests belong to the bandwidth agreement interface, so move it closer to them.

Reply:
that would create an import cycle

Reply:
that would cause an import cycle :/